lib/babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  presets: [
+    '@babel/preset-react',
+    '@babel/preset-typescript',
+    ['@babel/preset-env', { modules: false }],
+  ],
+  plugins: [
+    'optimize-clsx',
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
+    ['@babel/plugin-proposal-object-rest-spread', { loose: true }],
+    ['@babel/plugin-transform-runtime', { useESModules: true }],
```

Comment:
Looks like this affects all code. Better to add this and remove useESModules here
```
env: {
  esm: {
    plugins: [['@babel/plugin-transform-runtime', { useESModules: true }]]
  }
}
```

Reply:
This issue will be fixed
https://github.com/mui-org/material-ui-pickers/issues/1219